cmake/configs/default.cmake

```diff
@@ -112,7 +112,7 @@ if(_is_linux OR MINGW)
 elseif(IOS OR ANDROID)
   hunter_default_version(Qt VERSION 5.9.1-p0)
 else()
-  hunter_default_version(Qt VERSION 5.10.1)
+  hunter_default_version(Qt VERSION 5.11.1)
```

Comment:
Visual Studio build failed because of this error:
* https://bugreports.qt.io/browse/QTBUG-66652

Please add `if(MSVC)` here to use old Qt 5.10.1 version.

Reply:
Done.